lib/developer_portal/lib/liquid/drops/message.rb

```diff
@@ -7,7 +7,7 @@ class Message < Drops::Model
 
       desc "Returns the ID of the message."
```

Comment:
Probably this needs to be exended for clarity in the future. As it might be recipient id.

Reply:
Can we update just this comment for clarity. I'm not sure this is great but somehting along the lines of
```suggestion
      desc "Returns the ID of the model (depending on controller that is message or message recipient)."
```

Reply:
Otherwise good to merge! Thank you!

Reply:
@akostadinov I actually don't like this change :sweat_smile: The reason is that this description is for the users, not for the developers. It is seen in the admin portal at `/p/admin/liquid_docs`.

And for the end user this should be transparent, they probably don't know what a "controller" or what a "message recipient" is :upside_down_face: 

Maybe something more generic could be used, such as _"Returns the ID of the received or sent message"_.

Also, whenever we change the Liquid description, we need to run `bundle exec rake doc:liquid:generate` to make sure it gets updated in the corresponding docs.

Reply:
https://github.com/3scale/porta/pull/3675 :point_up: